Project Files/DLLSources/CvUnit.cpp

```diff
@@ -11476,61 +11476,58 @@ bool CvUnit::setProfession(ProfessionTypes eProfession, bool bForce, bool bRemov
 		return false;
 	}
 
-	if (getProfession() != eProfession)

```

Comment:
Following changes are basically removal of this condition, and subsequent change of indentation.